libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -699,4 +588,95 @@ public static Integer parseStringToNumber(String input) {
             return 0;
         }
     }
+
+    public static Map<String, ObligationStatusInfo> getProjectObligationToDisplay(
```

Comment:
`getProjectObligationToDisplay(-,-)`, `getOrganisationObligationToDisplay(-,-)`, `getComponentObligationToDisplay(-,-)`  - These methods contains mostly duplicated codes. Would be good if can be moved to a single method